tests/test_mpi.py

```diff
@@ -395,24 +395,28 @@ def test_ownership(self, shape, coords, points):
         assert list(ownership.keys()) == [grid.distributor.myrank]
 
     @pytest.mark.parallel(mode=4)
-    @pytest.mark.parametrize('coords,expected', [
-        ([(0.5, 0.5), (1.5, 2.5), (1.5, 1.5), (2.5, 1.5)], [[0.], [1.], [2.], [3.]]),
+    @pytest.mark.parametrize('coords,expected,expectedinds', [
+        ([(0.5, 0.5), (1.5, 2.5), (1.5, 1.5), (2.5, 1.5)], [[0.], [1.], [2.], [3.]], [(slice(0,1),),(slice(1,2),),(slice(2,3),),(slice(3,4),)]),
+        ([(1.5, 1.5),], [[],[],[],[0.]], [(slice(0,-1),),(slice(0,-1),),(slice(0,-1),),(slice(0,1),)])
```

Comment:
Looks good thanks. Can you check line length? We require <92 characters per line in CI.

You can run `flakke8 tests/test_mpi.py` to check (it should be installed as a devitio dependency).

Reply:
thanks for pointing that out! should now be resolved.  the line also needed some white space after commas :)